bin/main.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:io';
+import 'package:qc_apply/qc_apply.dart' as qc_apply;
+
+void main(List<String> arguments) {
```

Comment:
I'd usually include a few lines of inline docs to show how it's intended to be used. Even when it's obvious, an example can help the next person. We don't know how familiar they are with the language or how busy they may have been on another project.